src/pages/steps/AdditionalInformationStep.tsx

```diff
@@ -151,14 +164,31 @@ const AdditionalInformationStep = ({
     },
   });
 
+  const createOrganizerMutation = useCreateOrganizerMutation();
+
+  const addOrganizerToEventMutation = useAddOrganizerToEventMutation({
+    onSuccess: async () => {
+      await invalidateEventQuery('organizer');
```

Comment:
Do we need async / await here?

Reply:
I could write it as:

`onSuccess: () => invalidateEventQuery('organizer')`

I read that it is better to await promises explicitly. Later on the stacktrace of an error will be clearer, if it fails in that async function.

https://github.com/goldbergyoni/nodebestpractices#-212-always-await-promises-before-returning-to-avoid-a-partial-stacktrace

Reply:
I see. Interesting.